Mathlib/Order/Interval/Set/Monotone.lean

```diff
@@ -188,10 +188,18 @@ theorem strictMonoOn_Iic_of_lt_succ [SuccOrder α] [IsSuccArchimedean α] {n : 
   refine hψ _ (lt_of_lt_of_le ?_ hy)
   rwa [Function.iterate_succ', Function.comp_apply, lt_succ_iff_not_isMax]
 
+theorem strictMono_of_lt_succ [SuccOrder α] [IsSuccArchimedean α]
+    (hψ : ∀ m, ψ m < ψ (succ m)) : StrictMono ψ := fun _ _ h ↦
+  (strictMonoOn_Iic_of_lt_succ fun m _ ↦ hψ m) h.le le_rfl h
+
 theorem strictAntiOn_Iic_of_succ_lt [SuccOrder α] [IsSuccArchimedean α] {n : α}
     (hψ : ∀ m, m < n → ψ (succ m) < ψ m) : StrictAntiOn ψ (Set.Iic n) := fun i hi j hj hij =>
   @strictMonoOn_Iic_of_lt_succ α βᵒᵈ _ _ ψ _ _ n hψ i hi j hj hij
 
+theorem strictAnti_of_pred_lt [SuccOrder α] [IsSuccArchimedean α]
+    (hψ : ∀ m, ψ (succ m) < ψ m) : StrictAnti ψ := fun _ _ h ↦
```

Comment:
Should this not be ...succ_lt?

Reply:
Yeah, and also, we should have `pred` versions of these statements too.

Reply:
Wow sorry, I was clearly tired when I maintainer merged this. @YnirPaz, mind opening a follow-up PR?

Reply:
Wow sorry, these names are somehow short-circuiting my brain. #16813